### DISCUSSION TITLE
Welcome to SmartTaskbar Discussions!

### DISCUSSION THREAD

**Discussion by Oliviaophia** [ChanpleCai/SmartTaskbar#71] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pmoral86** [comment]
Is there any way to activate the taskbar over programs (not fullscreen) as in print? 
![Welcome_to_SmartTaskbar_Discussions!_·_Discussion_  - QA Paulo Moral - 07 02 22_08 40-PM](https://user-images.githubusercontent.com/63356646/152890513-f38aa306-e91b-457b-bcf5-6cb1feb55e5b.png)

**Comment by oasido** [comment]
Just found out about this project through Reddit, I am using StartAllBack along with this software. This is great!
Finally, there's an answer to Window's pretty bad taskbar management. I feel like I am way more focused without the taskbar showing on full-screen apps.
Thank you for this!

**Reply by Vectruz** [reply]
How exactly can i install this on my computer? I not found any tutorial or installation method.

**Comment by maccichu** [comment]
I also just found out about this and think it's fab.  My main use-case is when I'm switching between my small laptop screen and my large fixed monitors.  On my large monitors I'd be happy for the taskbar to show, but on the small screen it takes up too much space.
Appreciate this is stretching the product a little bit but it'd be awesome to have the auto mode look at (say) single vs multi- monitor mode, or if the active monitor is part of an extended desktop, or the screen size of the monitor (if screen size is greater than X then...) 

wishing you luck with your projects!!
